google/cloud/forseti/services/model/importer/importer.py

```diff
@@ -652,7 +661,7 @@ def _convert_appengine_resource(self, gae_resource):
             parent=parent)
 
         self.session.add(resource)
```

Comment:
If there is no is unique check on the resource, I think the duplicate key will occur again because the resource object has no reference to the unique identifier of the inventory resource (gae_resource.id), this is same for other resources. So what might happen is when 2 gae_resource with different resource id and same contents, it will generate the same resource object and insert it to the resource table.

Reply:
The unique constraint was added to the inventory, so this is explicitly no longer an issue when building the model.